test/integration/scheduler_perf/scheduler_perf_types.go

```diff
@@ -0,0 +1,28 @@
+package benchmark
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+		"k8s.io/kubernetes/pkg/api/v1"
+)
+
+// High Level Configuration for all predicates and priorities.
+type schedulerPerfConfig struct {
+	NodeAffinity *nodeAffinity
+	InterPodAffinity *interPodAffinity
+}
+
+// interPodAffinity priority configuration details.
+type interPodAffinity struct {
```

Comment:
@gmarek are you suggesting that these structs shouldn't carry any top level API types?   I guess that might make sense from the structures simplicity standpoint.  Otherwise I kind of like the type coupling because it  allows the compiler to find any changes that might happen. 

Just want to confirm Im getting the context  here.